libraries/models.py

```diff
@@ -259,7 +260,7 @@ def get_description(self, client, tag="develop"):
                 return body_content
 
         # If no content was found for any of the files
-        return None
+        return README_MISSING
```

Comment:
Hi Greg, thanks for the PR!
I think it would be better if get_description() continued to return None, and instead this was a check in the get_context_data function where this is called. Then we can specify where exactly we see a specific message on a more contextual basis. 

For an example, if we decide to check if the description is set in some cron job we can just check against None rather than having to compare against the constant, None being a more intuitive result for not set.

Reply:
Yep good call - I've made that change. Let me know if the new version makes sense to you. Tests still pass and manual testing looks the same.